### PR TITLE
Adds accountId + appName prefix to static unit buckets

### LINKED ISSUE #54: S3 Bucket for Static Unit is missing Account ID prefix
Static Unit creates a bucket for each site using the following bucket naming format: `static-website-<UNIT_ID>`. Since S3 bucket names are globally unique, deployment may fail if a bucket of the same name already exists.

A more appropriately namespaced alternative format is the following:
`<ACCOUNT_ID>-<APP_NAME>-<UNIT_ID>-sw`

Shortening `static-website` to `sw` gives the user back 12 characters to use in the unit ID which are at a premium now that we have two additional prefix components.

### PR DESCRIPTION
**Pull request by DavidSeptimus-Klotho**
This PR resolves #54 by replacing the `static-website` prefix with an `<accountId>-<appName>-` prefix to the buckets create for static units. Logic for shortening bucket names that are too long will be part of #18.
  
### Standard checks

- **Unit tests**: Any special considerations?
- **Docs**: Do we need to update any docs, internal or public?
- **Backwards compatibility**: Will this break existing apps? If so, what would be the extra work required to keep them working? will cause buckets created previously by static unit to be replaced

### CODE REVIEW

File: pkg/infra/pulumi_aws/index.ts.tmpl
Comment (jhsinger-klotho) [inline pkg/infra/pulumi_aws/index.ts.tmpl:146]: is the appname not a part of cloudLib? I thought it was and we could access it that way. If it isnt then we should make it so it is so that all information needed by our IaC code is available in cloudlib.

File: pkg/infra/pulumi_aws/iac/static_s3_website.ts
Comment (jhsinger-klotho) [inline pkg/infra/pulumi_aws/iac/static_s3_website.ts:17]: nit: I think its cleaner to just use the apply to get the string for the bucket name since thats all that we use it for and not have all the other code wrapped in the apply. Dont think it matters for functionality though

**Review by jhsinger-klotho — APPROVED** [review]
Nothing blocking, just small comments

File: pkg/infra/pulumi_aws/index.ts.tmpl
Comment (DavidSeptimus-Klotho) [reply pkg/infra/pulumi_aws/index.ts.tmpl:146]: It's not exported. This pattern is used elsewhere though I see a benefit to exporting it as a readonly property.

**Comment by DavidSeptimus-Klotho** [comment]
Holding off on merging this one for now since it may break existing CloudFront distributions

**Comment by gordon-klotho** [comment]
Should this be put back into draft until we're ready to merge?

**Comment by DavidSeptimus-Klotho** [comment]
This PR is no longer relevant.

**Pull request closed without merging**
